modules/apps/foundation/frontend-theme/theme3/src/js/main.js

```diff
@@ -0,0 +1,75 @@
+AUI().ready(
+
+	/*
+	This function gets loaded when all the HTML, not including the portlets, is
+	loaded.
+	*/
+function() {
+
+
+}
+);
+Liferay.Portlet.ready(
+
+	/*
+	This function gets loaded after each and every portlet on the page.
+
+	portletId: the current portlet's id
+	node: the Alloy Node object of the current portlet
+	*/
+
+	function(portletId, node) {
+	}
+);
+
+Liferay.on(
+	'allPortletsReady',
+
+	
+
+	function() {
+
+	
+	}
+);
+
+
+$(document).ready(function () {
+$(window).scroll(function() {
+	if($(document).scrollTop() > 950) {
+    	$('#navigation').addClass('hidenav');
+	}
+	else {
+	$('#navigation').removeClass('hidenav');
+	}
+	});
```

Comment:
Since this will fire on every window scroll, you should ```debounce``` or ```throttle``` this action so it does not fire on every scroll interaction.